modules/rest-compatibility/build.gradle

```diff
@@ -19,7 +19,12 @@
 
 esplugin {
   description 'Adds a compatiblity layer for the prior major versions REST API'
-  classname 'org.elasticsearch.rest.compat.RestCompatPlugin'
+  classname 'org.elasticsearch.compat.RestCompatPlugin'
+}
+
+dependencies {
```

Comment:
these two dependencies are to allow making compatible versions of Actions from these modules